src/parser/parser_nodes.nit

```diff
@@ -2456,6 +2456,15 @@ class ACallReassignExpr
 	super ASendReassignFormExpr
 end
 
+# A reference to a method with a captured receiver. eg. `&x.foo` or just `&foo` is self is captured.
```

Comment:
What do you think about adding the example from this PR description here so we don't loose it?

Reply:
:(